include/hermes/VM/PredefinedStrings.def

```diff
@@ -318,6 +318,8 @@ STR(multiline, "multiline")
 STR(unicode, "unicode")
 STR(sticky, "sticky")
 STR(dotAll, "dotAll")
+STR(indices, "indices")
```

Comment:
Are there two definitions needed `indicates` and `hasIndicates`?

Reply:
Yes, they are used [here](https://github.com/facebook/hermes/pull/968/files#diff-f4a4510e019a334caf037e7c736570bcc110022baa742adc6bdf0782ad11e70aR515), [here](https://github.com/facebook/hermes/pull/968/files#diff-f4a4510e019a334caf037e7c736570bcc110022baa742adc6bdf0782ad11e70aR123) and [here](https://github.com/facebook/hermes/pull/968/files#diff-f4a4510e019a334caf037e7c736570bcc110022baa742adc6bdf0782ad11e70aR2203).